### PR TITLE
Remove unwanted logs

### PR DESCRIPTION
**Pull request by kannappanr**
## Description
Some of the logs that are currently printed can be removed.
## Motivation and Context
We need to remove unwanted logs on the console.

## Regression
No

## How Has This Been Tested?
Run mint against minio started in erasure code mode

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added unit tests to cover my changes.
- [ ] I have added/updated functional tests in [mint](https://github.com/minio/mint). (If yes, add `mint` PR # here: )
- [x] All new and existing tests passed.

### CODE REVIEW

File: cmd/object-handlers.go [hunk 1]
```diff
@@ -266,15 +266,9 @@ func (api objectAPIHandlers) SelectObjectContentHandler(w http.ResponseWriter, r
 
 	// Executes the query on data-set
 	if err = s3select.Execute(w, s3s); err != nil {
-		logger.LogIf(ctx, err)
-
-	}
-
-	for k, v := range objInfo.UserDefined {
```
Comment (harshavardhana) [inline cmd/object-handlers.go]: We need this to be set for audit logging.

**Review by harshavardhana — CHANGES_REQUESTED** [review]

File: cmd/object-handlers.go (hunk 1 above)
Comment (kannappanr) [reply cmd/object-handlers.go]: done

**Review by harshavardhana — APPROVED** [review]

**Comment by minio-ops** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-tls.sh | :heavy_check_mark: |
| mint-xl.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :heavy_check_mark: |

**Review by vadmeste — APPROVED** [review]
LGTM

**Pull request merged**